src/main/kotlin/com/cognifide/gradle/aem/common/instance/service/repository/Node.kt

```diff
@@ -127,6 +128,16 @@ class Node(private val repository: Repository, val path: String) : Serializable
         return save(properties)
     }
 
+    /**
+     * Copies the node from current node to given path
+     */
+    fun copyTo(destination: String) = copy("${parent.path}/", destination)
```

Comment:
dislike trimming slashes

Reply:
generally API user should not take care about if it is needed or not. make your code defensive